### PR TITLE
Adding LL tags

### PR DESCRIPTION
**Pull request by estigma88**
Adding LL tags

### CODE REVIEW

File: tests/lib.rs [hunk 1]
```diff
@@ -478,116 +494,46 @@ fn parsing_binary_data_should_fail_cleanly() {
...
+#EXT-X-PART:URI=\"10_track1504_.m4s.1\",DURATION=0.510644,INDEPENDENT=YES
+#EXT-X-PART:URI=\"10_track1504_.m4s.2\",DURATION=0.487433,INDEPENDENT=YES
+#EXTINF:0.998467,
+10_track1504_.m4s
+#EXT-X-PART:URI=\"11_track1504_.m4s.1\",DURATION=0.510644,INDEPENDENT=YES
+#EXT-X-PRELOAD-HINT:TYPE=PART,URI=\"11_track1504_.m4s.2\"
+#EXT-X-RENDITION-REPORT:URI=\"playlist_1.m3u8\",LAST-MSN=1,LAST-PART=8
+";
+    let playlist_parsed = print_parse_and_create_playlist(&playlist_original);
+    assert_eq!(playlist_parsed, playlist_original);
```
Comment (JuanMorenoS) [inline tests/lib.rs:712]: You should add a test that builds the playlist from code

Comment (estigma88) [reply tests/lib.rs:712]: Why? the current test deserialize the String to Objects, and Objects to String, my feeling is that the test scope takes everything we need, from parsing to serializing

Comment (JuanMorenoS) [reply tests/lib.rs:712]: Ok, but that test serves as an example of how to create use the API, and you verify that the API is accessible and works.

Comment (estigma88) [reply tests/lib.rs:712]: Done

**Pull request merged**